sensorlib/config/simulated_sensor_config.yaml

```diff
@@ -10,11 +10,13 @@ simulated_sensor:
     max_distance_meters: 100
   detection_threshold_scaling_formula:
     nominal_hitpoint_detection_ratio_threshold: 0.6
-    hitpoint_detection_ratio_threshold_per_meter_change_rate: -0.0033
+    hitpoint_detection_ratio_threshold_per_meter_change_rate: 0.025
     adjustable_threshold_scaling_parameters:
       dropoff_rate: 0.01
   geometry_reassociation:
-    sample_count: 30
+    max_sample_count: 150
```

Comment:
Added max_sample_count because close objects can easily have hundreds of points.